<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="show_SPARE_scores_from_data_Btn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Show SPARE-* from data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="load_SPARE_model_Btn">
         <property name="text">
          <string>Load SPARE Model</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="compute_SPARE_scores_Btn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Compute SPARE-*</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>SPARE-* model details</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="load_other_model_Btn">
           <property name="text">
            <string>Load other model</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="add_to_dataframe_Btn">
           <property name="text">
            <string>Add to DataFrame</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
